scripts/unregistered_checks.py

```
@@ -45,35 +102,99 @@ def delete_torrents_and_files(client, config, use_delete_tags, delete_tags, dele
+        else:
+            # Only some torrents have unregistered trackers (cross-seeding)
+            cross_seeding_tag_hashes.extend(unregistered_hashes)
+            tag_counts[cross_seeding_tag] = tag_counts.get(cross_seeding_tag, 0) + len(unregistered_hashes)
+
+    # Apply tags in batches (2 API calls instead of N)
+    if not dry_run:
+        if default_tag_hashes:
+            try:
+                client.torrents_add_tags(torrent_hashes=default_tag_hashes, tags=[default_tag])
+                logging.info(f"Added tag '{default_tag}' to {len(default_tag_hashes)} torrents")
+            except Exception as e:
```

Ruff (0.14.3)

Do not catch blind exception: `Exception`

(BLE001)